README.md

```diff
@@ -2,7 +2,7 @@
 
 deploy tool for egg project.
 
-**Note: Windows is not supported**
+**Note: Windows is partially supported, see [#22](https://github.com/eggjs/egg-scripts/pull/22)**
```

Comment:
- egg guide docs of deploy is also need to updated.
- could raise anthoer PR to support `start` for win (if not `tail` instead, just ignore the checker at win)

Reply:
https://ci.appveyor.com/project/eggjs/egg-scripts

Reply:
PR for egg's docs, [#2788](https://github.com/eggjs/egg/pull/2788)